kong/templates/nginx_kong.lua

```diff
@@ -23,6 +23,10 @@ lua_shared_dict kong_db_cache               ${{MEM_CACHE_SIZE}};
 lua_shared_dict kong_db_cache_miss          12m;
 lua_shared_dict kong_secrets                5m;
 
+map "" $kong_request_id {
```

Comment:
This is mostly out of curiosity, but we should figure out if `map` is correct here. It probably is but we didn't use to in any   prior templates so we may want to spend some extra time to figure out any side-effects.

Reply:
https://nginx.org/en/docs/http/ngx_http_map_module.html

Reply:
looks like this is not needed after all. Defining log_format conditionally should make `set` enough and we can get rid of this.